app/models/election.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Election < ApplicationRecord
+  belongs_to :client
+  has_many :trustee
+  has_many :log_entry
```

Comment:
By convention this should this be in plural form:

```suggestion
  has_many :trustees
  has_many :log_entries
```